shared/controller-api/src/main/proto/Controller.proto

```diff
@@ -298,9 +298,22 @@ message RetentionPolicy {
         UNKNOWN = 0;
         TIME = 1;
         SIZE = 2;
+        CONSUMPTION = 3;
     }
     RetentionPolicyType retentionType = 1;
     int64 retentionParam = 2;
+    ConsumptionLimits consumptionLimits = 3;
+}
+
+message ConsumptionLimits {
+    enum ConsumptionLimitType {
+        UNKNOWN = 0;
```

Comment:
Here, UNKNOWN perhaps means we don;t set a limit for CBR. Could we call it NONE instead of UNKNOWN?

Reply:
UNKNOWN = 0 in protobuf allows future safety of enums evolution .. if reading protobuf messing written with new enum value using an old schema, protobuf sets new fields to unknown. 

if we want to support NONE, that should be a separate enum value. if you think we want to support "none" i could add that. but logically none is equivalent to providing limits as null.

Reply:
Yes we need to provide a NONE value, because a user could choose to not set any min/max limits..... true CBR...

Reply:
> UNKNOWN = 0 in protobuf allows future safety of enums evolution .. when reading a protobuf message using an older enum schema, any enum that is not understood is set to unknown.
> 
> if we want to support NONE, that should be a separate enum value. if you think we want to support "none" i could add that. but logically none is equivalent to providing limits as null.

Okay.